certified-connectors/Viafirma/apiDefinition.swagger.json

```diff
@@ -0,0 +1,341 @@
+{
+  "swagger": "2.0",
+    "info": {
+      "version": "1.0.0",
+      "title": "Viafirma",
+      "description": "Send sign requests to others, save your signed documents and much more thanks to Viafirma's connector",
+      "contact": {
+        "name": "Viafirma Support",
+        "url": "https://soporte.viafirma.com/",
+        "email": "helpdesk@viafirma.com"
+      }
+    },
+  "host": "services.viafirma.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/documents/api/v3/messages/": {
+      "post": {
+        "summary": "Send sign request through email",
+        "description": "Create and send a sign request through email",
+        "operationId": "SendSignRequest",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json\r",
+            "description": "Content-Type",
+            "x-ms-summary" : "Content-Type"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "*/*\r",
+            "description": "Accept",
+            "x-ms-summary" : "Accept"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "groupCode": {
+                  "type": "string",
+                  "description": "groupCode",
```

Comment:
@anfuca All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
@anfuca It is not updated as requested, Please let us know the reason for resolving this comment?